xml/http_servers.xml

```diff
@@ -1993,6 +1998,101 @@
     <param pos="0" name="os.vendor" value="HP"/>
     <param pos="0" name="os.family" value="TippingPoint"/>
     <param pos="0" name="os.device" value="IPS"/>
+    <param pos="0" name="hw.vendor" value="HP"/>
+    <param pos="0" name="hw.family" value="TippingPoint"/>
+    <param pos="0" name="hw.device" value="IPS"/>
+  </fingerprint>
+  <fingerprint pattern="^uc-httpd[ \/]([\d.]+)$">
+    <description>Xiongmai Tech uc-httpd</description>
+    <example service.version="1.0.0">uc-httpd 1.0.0</example>
+    <example service.version="1.0.0">uc-httpd/1.0.0</example>
+    <param pos="0" name="service.vendor" value="Xiongmai Tech"/>
+    <param pos="0" name="service.product" value="uc-httpd"/>
+    <param pos="1" name="service.version"/>
+  </fingerprint>
+  <fingerprint pattern="^micro_httpd$">
+    <description>ACME micro_httpd</description>
+    <example>micro_httpd</example>
+    <param pos="0" name="service.vendor" value="ACME"/>
+    <param pos="0" name="service.product" value="micro_httpd"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:acme:micro_httpd:-"/>
+  </fingerprint>
+  <fingerprint pattern="^mini_httpd$">
+    <description>ACME mini_httpd</description>
+    <example>mini_httpd</example>
+    <param pos="0" name="service.vendor" value="ACME"/>
+    <param pos="0" name="service.product" value="mini_httpd"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:acme:mini_httpd:-"/>
+  </fingerprint>
+  <fingerprint pattern="^LiteSpeed\/?(:?[\d.]+)?(?: \S+)?">
+    <description>LiteSpeed</description>
+    <example>LiteSpeed</example>
+    <example>LiteSpeed/5.2.8 Enterprise</example>
+    <param pos="0" name="service.vendor" value="LiteSpeed Technologies"/>
+    <param pos="0" name="service.product" value="LiteSpeed Web Server"/>
+    <param pos="1" name="service.version"/>
+  </fingerprint>
+  <fingerprint pattern="^IdeaWebServer\/v?([\d.]+)$">
+    <description>Idea Web Server</description>
+    <example>IdeaWebServer/0.83</example>
```

Comment:
Minor, no example, not much different from what is already there.

Reply:
Not sure I understand this comment.  Did you mean that the following two examples aren't different enough to warrant needing them?